### ISSUE TITLE
LiveLink Functionality in packaged game?

### ISSUE THREAD

**Issue by NiklasBothe** [ue4plugins/MobuLiveLink#14]
Hello, 

first of all, thank you for working on this amazing plug-in! 
Not being a coder, I was wondering why the plug-ins functionality doesn't carry over into a packaged build. This would be super-useful for our projects. Is something like this planned?

Thanks and best regards,
Niklas

**Comment by ekorkuti** [comment]
Yes, you can package the functionality into a game build but you need to implement the LiveLink connection in separate BP Actor.
Look at the documentation on how to achieve that, mainly at the Construct Message Bus Finder 
 and Connect to Provider nodes.
https://docs.unrealengine.com/en-US/Engine/Animation/LiveLinkPlugin/LiveLinkBlueprintComponent/index.html

**Comment by NiklasBothe** [comment]
Hey, 

thanks for the reply! I'm pretty sure I'm already doing that, it's just that the availabel message buses don't seem to show up in a packaged game. 

This is the blueprint I'm using, works fine in the editor but doesn't in the packaged game :x
This is a seperate object in the scene.
![grafik](https://user-images.githubusercontent.com/60506512/76312367-32e96700-62d3-11ea-9351-fb7c4343ca4e.png)

**Comment by ekorkuti** [comment]
To use the Live Link plugin in a packaged project, add the -Messaging flag to the command line when you start your project.

**Comment by NiklasBothe** [comment]
God bless you! 

Stay healthy out there and thanks for the help, means a lot!

**Comment by NateJ-Icon** [comment]
If someone comes here later, I'm using Unreal 5.0.2 and the -Messaging flag is no longer needed. The trick is to close the Unreal editor first! And if that doesn't work, try saving the Live Link profile as a preset in Project Settings.

**Comment by NiWoFu** [comment]
Hallo, sorry for digging up this thread, I tried the suggestion with get avaible providers in 5.2, so far in editor or as packed development build it works without any problem, but as soon as I try the shipping build and trigger the get avaible providers node the game crashes immediately with a fatal error window. Have someone also experienced such behavior?
